src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/controllers/v1/ReferenceDataController.kt

```diff
@@ -0,0 +1,172 @@
+package uk.gov.justice.digital.hmpps.hmppsintegrationapi.controllers.v1
+
+import ReferenceData
+import io.swagger.v3.oas.annotations.Hidden
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.media.Content
+import io.swagger.v3.oas.annotations.media.ExampleObject
+import io.swagger.v3.oas.annotations.media.Schema
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import org.springframework.boot.context.properties.EnableConfigurationProperties
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.config.AuthorisationConfig
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.hmpps.Response
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.services.ReferenceDataService
+
+@Hidden
+@RestController
+@EnableConfigurationProperties(AuthorisationConfig::class)
+@RequestMapping("/v1/reference-data")
```

Comment:
Can we make the URL "/v1/hmpps/reference-data" to make it clear this is HMPPS-wide reference data.

Reply:
Done